golang/cosmos/app/app.go

```diff
@@ -895,11 +895,6 @@ func unreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string) func(sdk.Conte
 			// Each CoreProposalStep runs sequentially, and can be constructed from
 			// one or more modules executing in parallel within the step.
 			CoreProposalSteps = []vm.CoreProposalStep{
-				/* upgrade-15 evals */
-				// Upgrade ZCF only
-				vm.CoreProposalStepForModules("@agoric/builders/scripts/vats/upgrade-zcf.js"),
```

Comment:
Since the failing test indicates that a ZCF upgrade is needed, this PR should at least maintain the ZCF upgrade core proposal. Since the plan for the next upgrade seem to upgrade both Zoe and ZCF, we could also do that instead in this PR.